src/app.js

```diff
@@ -68,6 +69,7 @@ module.exports = async () => {
     app.use('/v1/processors', processorsRouter);
     app.use('/v1/files', filesRouter);
     app.use('/v1/webhooks', webhooksRouter);
+    app.use('/v1/contexts', contextsRouter);
```

Comment:
Please add this api spec to
1. predator/docs/devguide/docs/swagger-docs.yaml
2. predator/docs/openapi3.yaml